### ISSUE TITLE
Audit components in Windows HC mode

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds#4469]
Need to complete an audit of all components in Windows HC mode

**Comment by mahoneycm** [comment]
The most important problems are the ones that lose functionality (missing buttons / clickable icons) or information (i.e. showing which button is selected vs not selected). After the audit and a meeting with Lauren on the SBA team, I found the following priority issues:

## Components with failing high contrast issues:

- [ ]  **Accordion** - Outline & expand/collapse icons missing for menu items (which makes it hard for users to tell it is an expandable accordion)
- [ ] **Banner** - Chevron missing (without those, users cannot tell they can expand the banner for more information)
- [ ] **Checkbox** - Checkboxes do not show in high contrast mode
- [ ] **Combo Box** - Chevron missing. Clear input icon on default value variant missing. (_also, we may want to add some best use instructions for `default value` variant. Instructions should never be put inside a form field if they are going to be lost once a user fills out the form field_)
- [ ] **Date Picker & Date Range Picker** - Calendar icon and month/year select arrows missing. For date range variants, you cannot see selected date range.
- [ ] **Footer** - Social media icons missing. Chevron icon missing from mobile view
- [x]  **Header** - Fixed in #4485 
- [ ] **Identiffier** - External website icons missing
- [ ] **Pagination** - Selected page should have a bold border or different color background. In high contrast it isn't possible to see what page the user is currently on.
- [ ] **Time Picker** - Missing chevron icon
